StyleCop.Analyzers/StyleCop.Analyzers.Test/LayoutRules/SA1514UnitTests.cs

```diff
@@ -1151,5 +1151,23 @@ public enum TestEnum
 
             await VerifyCSharpFixAsync(testCode, expected, fixedCode, CancellationToken.None).ConfigureAwait(false);
         }
+
+        [Fact]
+        [WorkItem(3849, "https://github.com/DotNetAnalyzers/StyleCopAnalyzers/issues/3849")]
+        public async Task TestClassInGlobalNamespaceAsync()
+        {
+            var testCode = @"
+/// <summary>
```

Comment:
❓ Do we have a test for this same case, but without a newline present at the start of the file?